content/g-sgov/g-sgov-umístění/g-sgov-umístění-glosář.ttl

```diff
@@ -39,7 +32,6 @@ g-sgov-umístění-pojem:místo a skos:Concept;
 g-sgov-umístění-pojem:má-zahraniční-adresu a skos:Concept;
   skos:broader <https://slovník.gov.cz/veřejný-sektor/pojem/má-lokalizaci-popisem>,
     z-sgov-pojem:typ-vztahu;
-  skos:definition "Zahraniční adresa textem. Adresy v zahraničí se řídí místními pravidly."@cs;
```

Comment:
@MichalMed @psiotwo toto chceme fakt odstranit ? Takovych veci je tady vic ... nekde se to presunulo ? (Nenasel jsem to v tomto commite).